proto/asn1-pdu-proto/asn1_proto_to_der.h

```diff
@@ -0,0 +1,74 @@
+#ifndef ASN1_PROTO_TO_DER_H_
+#define ASN1_PROTO_TO_DER_H_
+
+#include <cmath>
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <typeinfo>
+#include <vector>
```

Comment:
```suggestion
#include <stdint.h>

#include <string>
#include <vector>

```

Following with https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes , we place C headers, then C++ headers, then your project's .h , each with newline separations.

We remove the unused includes.